integration_tests/support/commands.js

```diff
@@ -0,0 +1,28 @@
+Cypress.Commands.add('signIn', (options = { failOnStatusCode: true }) => {
+  cy.request(`/`)
+  cy.task('getSignInUrl').then(url => cy.visit(url, options))
+})
+
+const getTable = (subject, options = {}) => {
```

Comment:
this is a pretty good function, was there a reason for moving away from typescript for this class?